src/Parser/Monk/Brewmaster/Modules/Items/StormstoutsLastGasp.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import { formatNumber } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+// multiplying the damage dealt by this factor gives the damage dealt by
+// shoulders
+//
+// note that this is NOT just 0.25
+const KEG_SMASH_DMG_FACTOR = 0.2;
+
+/**
+ * The Brewmaster legendary shoulders, Stormstout's Last Gasp.
+ *
+ * Grants an additional charge to Keg Smash, and increases Keg Smash
+ * damage by 25%. 
+ */ 
+class StormstoutsLastGasp extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  damage = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasShoulder(ITEMS.STORMSTOUTS_LAST_GASP.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if(event.ability.guid === SPELLS.KEG_SMASH.id) {
+      this.damage += KEG_SMASH_DMG_FACTOR * event.amount;
+    }
```

Comment:
You should consider using the getBonusDamage module for this type instead, it allows you to use the correct modifiers as constants for clarity

Reply:
Oh interesting, I'll check that out